src/reducers/walletConnectReducer.js

```diff
@@ -201,6 +210,12 @@ const walletConnectReducer = (
     case WALLETCONNECT_SESSION_RECEIVED:
       return { ...state, waitingForSession: false };
 
+    case TOGGLE_WALLET_CONNECT_PROMO_CARD:
+      return {
+        ...state,
+        promoCardCollapsed: action.collapsed,
```

Comment:
```suggestion
        promoCardCollapsed: action.payload.collapsed,
```

Reply:
Or just:
```suggestion
        promoCardCollapsed: action.payload,
```
to follow the common approach